nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/storage/PutGCSObject.java

```diff
@@ -194,6 +194,16 @@ public class PutGCSObject extends AbstractGCSProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor GZIPCONTENT = new PropertyDescriptor
+            .Builder().name("gzip.content")
+            .displayName("Allow GZIP Compression")
```

Comment:
Recommend adjusting the wording to use `Enabled` instead of `Allow` so the behavior is clear.
```suggestion
            .displayName("GZIP Compression Enabled")
```